app/controllers/profile_controller.rb

```diff
@@ -9,11 +9,9 @@ def index
     user = current_user if user.nil?
 
     # Determine how the profile should be displayed based on the user_id
-    @stories = if user == current_user
-                 paginate_stories(current_user)
-               elsif current_user.allies_by_status(:accepted).include?(user)
-                 paginate_stories(user)
-               end
+    if user == current_user || current_user.allies_by_status(:accepted).include?(user)
```

Comment:
This line is too long

Reply:
You should be able to use `current_user.mutual_allies?(user)` for the allies check